Docker/Dockerfile.environment-amd64

```diff
@@ -36,13 +36,16 @@ RUN wget 'https://boostorg.jfrog.io/artifactory/main/release/1.69.0/source/boost
 # Macintosh
 RUN mkdir -p /macintosh && \
     cd /macintosh && \
+    wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && /bin/bash ~/miniconda.sh -b && \
+    eval "$(/root/miniconda3/bin/conda shell.bash hook)" && \
     wget "https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u202-b08/OpenJDK8U-jdk_x64_mac_hotspot_8u202b08.tar.gz" && \
     tar axvf OpenJDK8U-jdk_x64_mac_hotspot_8u202b08.tar.gz && \
     rm -f OpenJDK8U-jdk_x64_mac_hotspot_8u202b08.tar.gz && \
-    wget "https://anaconda.org/conda-forge/libgdal/3.1.2/download/osx-64/libgdal-3.1.2-hd7bf8dc_4.tar.bz2" && \
-    mkdir -p gdal/3.1.2 && \
-    tar axvf libgdal-3.1.2-hd7bf8dc_4.tar.bz2 -C gdal/3.1.2 && \
-    rm -f libgdal-3.1.2-hd7bf8dc_4.tar.bz2
+    wget "https://anaconda.org/conda-forge/libgdal/3.7.2/download/osx-64/libgdal-3.7.2-h85f6614_6.conda" && \
+    mkdir -p gdal/3.7.2 && \
+    cph extract libgdal-3.7.2-h85f6614_6.conda --dest gdal/3.7.2 && \
```

Comment:
conda don't ship `tar.bz2` archives but uses it's own `tar.bz2`-like packaging => so I had to install conda to extract it 🤦 

https://github.com/conda/conda-package-handling

Reply:
Maybe we could turn these version strings into `buildarg`s

Reply:
I went mb a bit too far :D https://github.com/geotrellis/gdal-warp-bindings/pull/120/commits/d0f582888196164c35b9c0662b1bce3e0341eb6b#diff-93c714fe4d67d93f9c74b0413dc5d5b7975e04602a2ce531ac5f5c4034e5f3fcR4-R9 

Two visible cons: MacOS and Windows GDAL versions are not really versions, but achive names. Still not a too bad result it looks like. For linux builds its a really nice parametrization.